website/docs/r/purview_account.html.markdown

```diff
@@ -39,6 +39,10 @@ The following arguments are supported:
 
 * `public_network_enabled` - (Optional) Should the Purview Account be visible to the public network? Defaults to `true`.
 
+* `managed_resource_group_name` - (Optional) The name of the Resource Group where Purview Account creates the managed resources.
```

Comment:
We could remove the note and change the description:

```suggestion
* `managed_resource_group_name` - (Optional) The name which should be used for the new Resource Group where Purview Account creates the managed resources. Changing this forces a new Purview Account to be created.
```

Reply:
Description updated